x-pack/plugins/drilldowns/public/types.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { AdvancedUiActionsActionFactoryDefinition as ActionFactoryDefinition } from '../../advanced_ui_actions/public';
+
+/**
+ * This is a convenience interface to register a drilldown. Drilldown has
+ * ability to collect configuration from user. Once drilldown is executed it
+ * receives the collected information together with the context of the
+ * user's interaction.
+ *
+ * `Config` is a serializable object containing the configuration that the
+ * drilldown is able to collect using UI.
+ *
+ * `PlaceContext` is an object that the app that opens drilldown management
+ * flyout provides to the React component, specifying the contextual information
+ * about that app. For example, on Dashboard app this context contains
+ * information about the current embeddable and dashboard.
+ *
+ * `ExecutionContext` is an object created in response to user's interaction
+ * and provided to the `execute` function of the drilldown. This object contains
+ * information about the action user performed.
+ */
+export interface DrilldownDefinition<
```

Comment:
Looks like this file is now a duplicate and should be deleted

Reply:
Done.